### PR TITLE
ab: wb, band sizes

### PR DESCRIPTION
**Pull request by annagav**

### CODE REVIEW

File: instructor/migrations/0007_auto_20151105_0946.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```
Comment (noisecapella) [inline instructor/migrations/0007_auto_20151105_0946.py]: Can you rename this migration?

File: instructor/models.py [hunk 2]
```diff
@@ -128,18 +128,20 @@ class WesternBlotAntibody(models.Model):
...
     antibody = models.ForeignKey(WesternBlotAntibody, related_name='bands')
     strain_protocol = models.ForeignKey(StrainTreatment, related_name='bands')
-    wcl_weight = models.FloatField(default=0.00)
-    wcl_intensity = models.FloatField(default=0.00)
-    nuc_weight = models.FloatField(default=0.00)
-    nuc_intensity = models.FloatField(default=0.00)
-    cyto_weight = models.FloatField(default=0.00)
-    cyto_intensity = models.FloatField(default=0.00)
-    is_background = models.BooleanField(default=False)
+    intensity = models.FloatField(default=0.00)
```
Comment (noisecapella) [inline instructor/models.py]: It seems a bit cleaner to use `0.0` instead of `0.00` but no strong opinion here

**Comment by noisecapella** [comment]
Functionality works fine!

File: instructor/migrations/0007_auto_20151105_0946.py (hunk 1 above)
Comment (annagav) [inline instructor/migrations/0007_auto_20151105_0946.py]: Fixed

File: instructor/models.py (hunk 2 above)
Comment (annagav) [inline instructor/models.py]: Fixed

**Comment by noisecapella** [comment]
:+1:

**Pull request merged**